packages/downloadable-blocks/src/components/block-icon/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Path, Icon, SVG } from '@wordpress/components';
+
+export default function BlockIcon( { icon, showColors = false, className } ) {
```

Comment:
Isn't `BlockIcon` duplicated from another package?

Reply:
Good point. I'm reusing the `BlockIcon` from `block-editor` now.